README.md

```diff
@@ -23,6 +23,7 @@
 
 1. Run `bundle install` to install the gem dependencies
 2. Run `bundle exec rails db:setup` to create a development and testing database
+   1. MacOS: If you get an error `FATAL: role "postgres" does not exist` run `createuser -s postgres` in your terminal
```

Comment:
I think this applies to postgres regard less of OS

@timabell for input

Reply:
Ok, MacOS removed :)